api/src/pcapi/core/educational/api/stock.py

```diff
@@ -82,7 +83,21 @@ def edit_collective_stock(
     if booking_limit is None:
         check_booking_limit_datetime = stock.bookingLimitDatetime
 
-    offer_validation.check_booking_limit_datetime(stock, check_beginning, check_booking_limit_datetime)
+    booking_status = [booking.status for booking in stock.collectiveBookings]
+    if not stock_data.get("beginningDatetime") and not stock_data.get("bookingLimitDatetime"):
+        price = updatable_fields.get("price")
+        if CollectiveBookingStatus.USED in booking_status:
```

Comment:
```suggestion
        if booking and booking.status == CollectiveBookingStatus.USED:
```

Reply:
`if CollectiveBookingStatus.USED in current_booking_status:`